### DISCUSSION TITLE
Update codes to have a consistancy

### DISCUSSION THREAD

**Discussion by baseplate-admin** [facelessuser/pymdown-extensions#2216] [Ideas]
It seems that old syntax is used here :

1. https://github.com/facelessuser/pymdown-extensions/blob/5dd162a2258d55cadf3a47e45f1ea0cb6bb23a37/pymdownx/highlight.py#L484
2. https://github.com/facelessuser/pymdown-extensions/blob/5dd162a2258d55cadf3a47e45f1ea0cb6bb23a37/pymdownx/highlight.py#L542

It should be changed to `super().__init__(*args, *kwargs)`

**Comment by facelessuser** [comment]
Sure. A PR is welcome.

**Reply by facelessuser** [reply]
https://github.com/facelessuser/pymdown-extensions/pull/2218 modernizes `super()` usage
